os/arch/arm/src/armv7-a/arm_mmu.c

```diff
@@ -104,6 +108,9 @@ static void mmu_set_flags(uint32_t *val, bool ro, bool exec, uint8_t isL1)
  ****************************************************************************/
 
 #ifndef CONFIG_ARCH_ROMPGTABLE
```

Comment:
This is the arm architecture code. It looks strange with chip specific code here. @kishore-sn how do you think?

Reply:
True.. We could remove the check for AMEBASMART chip and keep the macro for placing it in ram. 
If the macro is used in ld file, then it will get placed in the ram.

Reply:
> True.. We could remove the check for AMEBASMART chip and keep the macro for placing it in ram. If the macro is used in ld file, then it will get placed in the ram.

I think the checking is still needed, as we will not expect all cortex-A chipset face similar issue, and the section config might varies between too,

Reply:
How about you just add the whole arm_mmu.o in your psram section in linker script?

Reply:
Good idea, done changing~